src/test/groovy/org/prebid/server/functional/tests/BidValidationSpec.groovy

```diff
@@ -212,4 +212,34 @@ class BidValidationSpec extends BaseSpec {
         def metrics = defaultPbsService.sendCollectedMetricsRequest()
         assert metrics["adapter.generic.requests.bid_validation"] == initialMetricValue + 1
     }
+
+    def "PBS should return error when between refers to unknown bidder alias"() {
+        given: "Default basic BidRequest"
+        def bidRequest = BidRequest.defaultBidRequest
+        bidRequest.ext.prebid.aliasgvlids = ["between": 1]
+        bidRequest.ext.prebid.aliases = ["appnexus": GENERIC]
+
+        when: "Sending auction request to PBS"
+        defaultPbsService.sendAuctionRequest(bidRequest)
+
+        then: "Request should fail with error"
+        def exception = thrown(PrebidServerException)
+        assert exception.responseBody.contains("Invalid request format: request.ext.prebid.aliasgvlids. " +
+                "vendorId between refers to unknown bidder alias: 1")
+    }
+
+    def "PBS should return error when aliasgvlids value lowest that one"() {
```

Comment:
`PBS should return an error when GVL ID alias  value is lower that one`

Reply:
Same, that comment wasn't answered here nor resolved in the code.